app/src/androidTest/java/com/example/android/courtcounter/tests/CourtCounterTest.kt

```diff
@@ -7,17 +7,17 @@ import androidx.test.espresso.matcher.ViewMatchers.*
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.example.android.courtcounter.MainActivity
 import com.example.android.courtcounter.R
-import com.example.android.courtcounter.Utils.BaseTest
+import com.example.android.courtcounter.utils.BaseTest
 import org.hamcrest.Matchers.allOf
 import org.junit.Test
 import org.junit.runner.RunWith
 
 //type out the run with line and it will import the required parts
 @RunWith(AndroidJUnit4::class)
-class myFirstTest: BaseTest() {
+class MyFirstTest: BaseTest() {
 
     @Test
-    fun checkTeamAPoints() {
+    fun CheckTeamAPoints() {
```

Comment:
This was actually fine as it was, I should maybe have been clearer, I was only talking about class names for the capitalisation, for functions/methods I think its standard to have the format you already had